src/intelli/graphconnectionmodel.h

```diff
@@ -1019,6 +1020,12 @@ class ConnectionModel_t
 
     bool containts(key_type const& key) const { return m_data.contains(key); }
 
+    bool empty() const { return m_data.empty(); }
+    bool isEmpty() const { return m_data.empty(); }
+
+    size_type size() const { return m_data.size(); }
```

Comment:
Why size_type and not size_t?

Reply:
In this class I use QHash. Any container that should be compatible with the STL should define a `size_type` variable. Here I use forwarded `QHash::size_type` and I thought why not use it here as well.